src/frontend/js/contexts/OrderContext/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { createContext, PropsWithChildren, useContext, useEffect, useMemo, useState } from 'react';
```

Comment:
`useOrderContext` is used only once, is it necessary to create a context for it ?

Reply:
Absolutely, to avoid going through multiple components of the same prop.